src/JustSaying/Messaging/Channels/Multiplexer/MergingMultiplexer.cs

```diff
@@ -65,7 +65,7 @@ private async Task RunImplAsync()
                     "Starting up channel multiplexer with a queue capacity of {Capacity}",
                     _channelCapacity);
 
-                ChannelExtensions.Merge(_readers, _targetChannel.Writer, _stoppingToken);
+                var _ = ChannelExtensions.Merge(_readers, _targetChannel.Writer, _stoppingToken);
```

Comment:
Drop the `var`, so it's a discard rather than a variable named `_`

Reply:
Done, didn't know this was possible with discards actually 🙂